include/seqan/sequence/string_packed.h

```diff
@@ -579,10 +583,74 @@ resize(
         clear(host(me));
         return 0;
     }
-    TStringSize max_length = (resize(host(me), TTraits::toHostLength(new_length) + 1, tag) - 1) * TTraits::VALUES_PER_HOST_VALUE;
-    if ((TStringSize)new_length > max_length)
-        new_length = max_length;
-    return front(host(me)).i = new_length;
+
+    TStringSize _new_length = new_length;
```

Comment:
There was a problem with using the enum value in the function below, so the cast did not work (but assignment did). With the last change this could be removed again...